src/core/Akka.Cluster.Tests.MultiNode/TransitionSpec.cs

```diff
@@ -256,6 +259,7 @@ private void A_Cluster_must_perform_correct_transitions_when_third_joins_second(
             EnterBarrier("convergence-joining-3");
 
             var leader12 = Leader(_config.First, _config.Second);
+            Log.Debug("Leader: {0}", leader12);
```

Comment:
Used this to help expose the bug; won't log anything by default since we run with `DebugConfig = false`